person.rb

```diff
@@ -17,4 +18,8 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def validate_name
+    @name = @corrector.correct.name(@name)
+  end
```

Comment:
- [ ] in this method you are referring to the `@corrector` instance variable, however, this instance variable was never created in the `initialize` method. which will look something like:
```
 @corrector = Corrector.new()
```